src/lib/adapters/spark.js

```diff
@@ -28,16 +28,25 @@ function SparkAdapter(robot) {
 
 util.inherits(SparkAdapter, SlackLikeAdapter);
 
-SparkAdapter.prototype.postData = function(data) {
+SparkAdapter.prototype.postData = function (data) {
   var self = this;
-
-  var recipient, split_message, formatted_message,
-      text = "";
+  var recipient, split_message, formatted_message, envelope,
```

Comment:
```suggestion
  var split_message, formatted_message, envelope,
```

Remove, since recipient isn't used anywhere anymore - see comment below.

Reply:
removed